src/domain/payments/index.types.d.ts

```diff
@@ -82,7 +82,7 @@ type LightningPaymentFlowBuilder<S extends WalletCurrency> = {
     description,
   }: {
     uncheckedAmount: number
-    description: string | null
+    description: string
```

Comment:
@bodymindarts I think the problem here is that the type for `withoutInvoice` is defined in 2 places.